### DISCUSSION TITLE
Copilot is not providing suggestions

### DISCUSSION THREAD

**Discussion by theyashshahs** [community/community#7528] [Copilot Conversations]
For the two days, the copilot is not working
![image](https://user-images.githubusercontent.com/42682370/141063322-278b5993-0cf6-4b8d-b75f-c33debec9fc6.png)

It is showing internal server error 500 😢 

GitHub copilot version: `v1.7.3733`
VS Code version: `1.62.1 (user setup)`

Anyone else facing the same issue?

**Comment by Megasware128** [comment]
Yes started having the same issue since an hour or so. Doesn't matter if I'm using a new machine or a different project. Just get 500 errors. I guess it is user related because on Twitter it seems to work for most people (like me couple of hours ago)

**Reply by Megasware128** [reply]
One guess would be that I/we hit some kind of usage limit if that is a thing 😅
@theyashshahs do you remember what your last working suggestion was? Because mine was HTML related.

**Reply by Megasware128** [reply]
Ok. Mine just started working again so I should have been more patient I guess 😅

**Reply by theyashshahs** [reply]
I downgrade the copilot version and it is working again.

**Reply by theyashshahs** [reply]
Maybe the version has some bug, I guess

**Reply by madsh93** [reply]
Can confirm that downgrading to 1.6.* worked for me. Seems 1.7.* did something to cause this.
